tests/e2e_tests/extended_e2e_test_cases.yaml

```diff
@@ -0,0 +1,1323 @@
+test_cases:
+# Explanation of format:
+#  - name: some_name
+#    config:
+#      # add any non-default configuration here
+#    raw_data:
+#    - source_url: https://source_of_file or folder
+#    library:
+#    - source_url: https://source_of_file
+#    fasta:
+#    - source_url: https://source_of_file
+#    metrics:
+#      # possible selections are all subclasses of the calc_metrics.py:Metrics class
+
+
+# Astral test cases
+## Targeted test cases
+  - name: astral_Wh80_targeted
+    config:
+      library_prediction:
+        predict: true
+        fixed_modifications: 'Carbamidomethyl@C'
+        variable_modifications: 'Oxidation@M;Acetyl@Protein N-term'
+        max_var_mod_num: 2
+        missed_cleavages: 1
+        precursor_mz:
+          - 380
+          - 980
+        nce: 25
+        instrument: Lumos
+      search:
+        target_num_candidates: 3
+        target_ms1_tolerance: 4
+        target_ms2_tolerance: 6
+        target_rt_tolerance: 0.1
+      search_initial:
+        initial_num_candidates: 1
+        initial_ms1_tolerance: 50
+        initial_ms2_tolerance: 50
+        initial_rt_tolerance: 0.5
+      search_output:
+        peptide_level_lfq: true
+        precursor_level_lfq: true
+    fasta:
+      - source_url: "//samba-pool-mann-pub.biochem.mpg.de/pool-mann-projects/despard/hpo/rt_optimizer_tuning_2024_08_22/libraries/2024_02_21_ecoli.fasta"
+      - source_url: "//samba-pool-mann-pub.biochem.mpg.de/pool-mann-projects/despard/hpo/rt_optimizer_tuning_2024_08_22/libraries/2024_02_21_human.fasta"
+      - source_url: "//samba-pool-mann-pub.biochem.mpg.de/pool-mann-projects/despard/hpo/rt_optimizer_tuning_2024_08_22/libraries/2024_02_21_yeast.fasta"
+    raw_data:
```

Comment:
could you add some information about the dataset here? (e.g. gradient length, sample composition, ?)